### ISSUE TITLE
DNK 3090 product ID seems wrong

### ISSUE THREAD

**Issue by cmsj** [ianmarmour/nvidia-clerk#163]
Pretty sure I have the latest release:

```
./nvidia-clerk-darwin-5 -desktop -remote -model 3090 -region DNK                                                                 ─╯
2020/10/01 09:37:54 Product ID: 5438793500
2020/10/01 09:37:54 Product Name: NVIDIA GEFORCE RTX NVLINK BRIDGE 4-SLOT FOR RTX 3090 - FI/DK
```

**Comment by redstone2019** [comment]
yes it is wrong https://api-prod.nvidia.com/direct-sales-shop/DR/products/en_gb/DKK/5438793500

**Comment by redstone2019** [comment]
fixed in PR #165 https://api-prod.nvidia.com/direct-sales-shop/DR/products/en_gb/DKK/5438793200
